app/services/users.js

```diff
@@ -0,0 +1,18 @@
+const { ValidationError } = require('sequelize');
+const { hash } = require('bcrypt');
+
+const { User } = require('../models');
+const { sequelizeValidationError } = require('../errors');
+
+const withHashedPassword = ({ password, ...userParams }) => {
+  const saltRounds = 10;
+  return hash(password, saltRounds).then(hashedPassword => ({ ...userParams, password: hashedPassword }));
+};
```

Comment:
Password hashing should not be in the user service. Which do you think is a better place?
Another comment: remember that functions should do, ideally, one thing. The hashing function only cares about the password. It could work without the rest of the user params.